### ISSUE TITLE
Opening datepicker inside a modal causes flickering!

### ISSUE THREAD

**Issue by great2lrn** [react-native-datetimepicker/datetimepicker#870]
This happens with @react-navigation/native using screen **_presentation: "modal"_**

When running a production installation of my app on my **iPhone SE (3rd gen),** opening the datepicker inside of the screen makes the whole screen jump (flicker).

However this does not happen when running the app using the expo go on a real device nor on a simulator. This only happens when the app binaries has been built, it also only happens on phones with physical home buttons.

[link to repro](https://snack.expo.dev/tNW_W7MIxPzMdhkw9jwjD)

### Video of current behaviour.
https://github.com/react-native-datetimepicker/datetimepicker/assets/96067238/1ecc5717-4331-43eb-887f-185ec20435c0

### Video of expected behaviour.

https://github.com/react-native-datetimepicker/datetimepicker/assets/96067238/f9fab734-ad07-4240-8a8f-5523330b430c

**Comment by ErMapsh** [comment]
did you found any solution on it

**Comment by great2lrn** [comment]
> did you found any solution on it

Nope, abandoned the whole idea of a modal unfortunately because of this.

**Comment by lbxa** [comment]
@great2lrn bottom sheet works fine :D
